tests/testthat/test-quitteSort.R

```diff
@@ -0,0 +1,22 @@
+context('quitteSort()')
+
+test_that('quitteSort works', {
+  for (qe in list(quitte_example_dataAR6, quitte_example_data)) {
+    qe <- quitteSort(as.quitte(qe, na.rm = TRUE))
+    expect_identical(qe, quitteSort(qe))
```

Comment:
So, this tests whether `quitteSort()` sorts based on the right ascension of Mars?